### PR TITLE
Setup to publish package

### PR DESCRIPTION
**Pull request by louiscoste**
I need this package to be published because this package is needed in `document-builder-worker` that does the npm install in the `Dockerfile`. Currently to get this package it needs to install git to get it. I think it is a bad idea to get install git for the single purpose of getting this package when we could simply publish this package

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
-  "name": "bunyan-firehose",
-  "version": "1.1.0",
+  "name": "@d2l/bunyan-firehose",
+  "version": "1.0.7",
```
Comment (TomMichielsD2L) [inline package.json:3]: Why did we move down in version?

File: README.md [hunk 2]
```diff
@@ -20,7 +20,7 @@ AWS Firehose extension for the Bunyan logger.
 ## Installing
 
 ```bash
-npm i github:Brightspace/bunyan-firehose
+npm i @d2l/bunyan-firehose
```
Comment (TomMichielsD2L) [inline README.md:23]: Do we break anything by moving to the d2l scope? Are there many repos that need to be updated to point to this new place?

File: package.json (hunk 1 above)
Comment (louiscoste) [reply package.json:3]: to match the version in github. I have no idea why it was at `1.1.0` on the `package.json`

File: README.md (hunk 2 above)
Comment (louiscoste) [reply README.md:23]: Currently the only way to get this package is by doing a direct reference through github using `github:Brightspace/bunyan-firehose`. This method will still work but what this PR is going to allow is to reference this PR through npm by using `@d2l/bunyan-firehose`. So all the current repos should still work. I also believe there is only one repo using this (but I might be wrong about that)

Comment (louiscoste) [reply README.md:23]: Seems it is the case: https://github.com/search?q=org%3ABrightspace+bunyan-firehose&type=code

**Review by TomMichielsD2L — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (quinnbudan) [reply README.md:23]: It won't change imports at all right? You can still just import `bunyan-firehose`?

**Review by quinnbudan — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (louiscoste) [reply README.md:23]: At the moment we import by writting in `package.json` `"bunyan-firehose": "github:Brightspace/bunyan-firehose#v1.0.7"` which will still work. With this change we will be able to also do `"@d2l/bunyan-firehose": "1.0.7"`

**Review by TomMichielsD2L — APPROVED** [review]

**Pull request merged**